src/commands/gcr-auth.yml

```diff
@@ -51,7 +51,7 @@ steps:
         # configure Docker to use gcloud as a credential helper
         mkdir -p "$HOME/.docker"
 
-        if [[ <<parameters.registry-url>> == *docker.pkg.dev ]]; then
+        if [[ "<<parameters.registry-url>>" == "*docker.pkg.dev" ]]; then
```

Comment:
I believe this should be:

if [[ "<<parameters.registry-url>>" == *"docker.pkg.dev" ]]; then

So bash does a wildcard comparison instead of comparing against the "*docker.pkg.dev" string itself

Reply:
Fixed on #51